target/scripts/build/packages.sh

```diff
@@ -82,12 +77,19 @@ function _install_packages() {
     opendmarc libsasl2-modules sasl2-bin
   )
 
+  # `bind9-dnsutils` provides the `dig` command
+  # `iputils-ping` provides the `ping` command
```

Comment:
I figured that `ping` was a bit more self-explanatory, but not fussed either way 😛